pkg/metrics/listener.go

```diff
@@ -21,11 +21,9 @@ import (
 	"net"
 )
 
-// DefaultBindAddress sets the default bind address for the metrics
-// listener
-// The metrics is off by default.
-// TODO: Flip the default by changing DefaultBindAddress back to ":8080" in the v0.2.0.
-var DefaultBindAddress = "0"
+// DefaultBindAddress sets the default bind address for the metrics listener
+// The metrics is on by default.
+var DefaultBindAddress = ":8080"
```

Comment:
We should update the documentation for manager.Options https://godoc.org/sigs.k8s.io/controller-runtime/pkg/manager#Options explaining how to turn off or get the previous default behavior.